### PR TITLE
feat(containeranalysis): generate library

### LINKED ISSUE #8108: Container Analysis API: `containeranalysis.googleapis.com`

### PR DESCRIPTION
**Pull request by coryan**
Fixes #8108

---
This change is [Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/8129)

### CODE REVIEW

File: google/cloud/containeranalysis/README.md [hunk 1]
```diff
@@ -3,7 +3,9 @@
 :construction:
 
 This directory contains an idiomatic C++ client library for the
-[Container Analysis API][cloud-service-docs], a service to An implementation of the Grafeas API, which stores, and enables querying and retrieval of critical metadata about all of your software artifacts.
+[Container Analysis API][cloud-service-docs]. An implementation of the
```
Comment (devbww) [inline google/cloud/containeranalysis/README.md:6]: s/. An/, an/

File: google/cloud/containeranalysis/doc/main.dox [hunk 2]
```diff
@@ -2,8 +2,10 @@
 
 @mainpage Container Analysis API C++ Client Library
 
-An idiomatic C++ client library for the [Container Analysis API][cloud-service-docs], a service
-to An implementation of the Grafeas API, which stores, and enables querying and retrieval of critical metadata about all of your software artifacts.
+An idiomatic C++ client library for the [Container Analysis API][cloud-service-docs].
+An implementation of the[Grafeas API](https://grafeas.io), which stores,
```
Comment (devbww) [inline google/cloud/containeranalysis/doc/main.dox:6]: s/. An/, an/

**Review by devbww — APPROVED** [review]

File: google/cloud/containeranalysis/README.md (hunk 1 above)
Comment (coryan) [reply google/cloud/containeranalysis/README.md]: Fixed.

File: google/cloud/containeranalysis/doc/main.dox (hunk 2 above)
Comment (coryan) [reply google/cloud/containeranalysis/doc/main.dox]: Fixed.

**Pull request merged**